config/feature.js

```diff
@@ -15,6 +15,10 @@ function isEnabled(key) {
 }
 
 function isDevelopment() {
+  if(isFirefoxPanel()) {
+    // Default to production if compiling for the Firefox panel
+    return process.env.NODE_ENV === "development";
+  }
```

Comment:
The panel before was still pulling in the development config, so it was bundling in hot module reloading and such.